build-logic/build.gradle

```diff
@@ -20,7 +20,7 @@ repositories {
 
 dependencies {
 	implementation "org.quiltmc:loom:1.0.+"
-	implementation "net.kyori:indra-git:3.0.0"
+	implementation "net.kyori:indra-git:3.0.1"
```

Comment:
whaaaaaaaaa??? wdym they updated already?
(dismiss this, it's just an observation :P)

Reply:
Everything gets updated faster than people can blink lol.